src/apps/activity_flows/crud/flow_history.py

```diff
@@ -57,3 +59,18 @@ async def retrieve_by_applet_ids(self, applet_versions: list[str]) -> list[Activ
         )
         db_result = await self._execute(query)
         return db_result.scalars().all()
+
+    async def get_by_applet_id_for_summary(self, applet_id: uuid.UUID) -> list[ActivityFlowHistoriesSchema]:
+        query: Query = select(ActivityFlowHistoriesSchema)
```

Comment:
Please check with BA the logic of what flow versions should be displayed. This query selects all history entities (last version) even if it's not already in the applet